### ISSUE TITLE
How to print (or reprint) all errors/warnings at the end?

### ISSUE THREAD

**Issue by jordwalke** [reasonml/rebel#61]
I have a ton of log output while compiling and I saw some warnings go by quickly. Is there a way to get all the warnings to be printed at the end? (Or maybe just reprinted again)?

**Comment by vramana** [comment]
> Jared Forsyth @jaredly Sep 29 19:40
> @vramana @chenglou one annoying thing about rebel is that it doesn't know when to quit. If you have an error in one file, then it spams you with errors about all the files that depend on that module  saying it doesn't exist (because it couldn't compile). Would be much better to just fail fast

@jordwalke Will bailing out early help?

**Comment by jordwalke** [comment]
Yes but then you lose other benefits. It's generally good to compile everything as much as you can. And I like the incremental delivery of warnings and errors because I can begin investigating while the rest is compiled. But a nice repeated summary at the end would be nice.

**Comment by chenglou** [comment]
Jenga does. At least it has a flag.

**Comment by vramana** [comment]
@chenglou Are you referring to `stop-on-first-error` flag?

**Comment by jaredly** [comment]
I'm ok with not stopping on first error, as long as the output is at all readable :P

**Comment by jordwalke** [comment]
Yeah, I like not stopping on the first error because while I'm debugging the first error, it can continue to build other unrelated things. Then when I fix the issue in the first error, it will compile the final executable faster. It's just that it's tough to spot the errors when they go flying by.
